scripts/dev/gha_regressions.py

```diff
@@ -505,6 +507,10 @@ def check_all_regressions(self, base_testfiles: Path, mod_testfiles: Path, bundl
             except Exception as e:
                 any_diffs = True
                 print(f"Regression run *failed* trying to process file: {baseline.name}; reason: {e}")
+                if not backtrace_shown:
+                    print("Traceback shown once:")
+                    print_exc()
+                    backtrace_shown = True
```

Comment:
When the regression script has an exception, this will print a traceback which will save me multiple hours of trying to debug and reproduce just to get the call stack of the error.  And it only shows it one time to avoid bloating the output even more.